app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -29,10 +32,8 @@ const DeleteAlert = ({
     <Alert
       isOpen
       isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      message="Are you sure you want to delete the note? This action cannot be undone."
+      title={`Delete Note`}
```

Comment:
Go through the Slack thread where I have already pointed out several review comments regarding the same.